### ISSUE TITLE
Auto-scan option!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kaicastledine/armitage#78]
```
Auto-scan function to the host menu.
Auto detects the IP range, and scans with optimal settings. (Whatever you think 
best, e.g OS detect, ports etc.)
Then initiates a metasploit scan.

Chris
```

Original issue reported on code.google.com by `chrisvel...@gmail.com` on 14 Dec 2011 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
I'm going to close this ticket because I'm working on a project to make 
Armitage extensible. Once it is complete, this will allow tools like Nessus to 
be integrated without putting it into the core of the tool. It's my intention 
to keep Armitage as light as possible while pushing additional 
features/integrations off to extensions.
```

Original comment by `rsmu...@gmail.com` on 15 Dec 2011 at 4:22
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Wow sounds fantastic, any guess-timate on when it will be released?
```

Original comment by `chrisvel...@gmail.com` on 16 Dec 2011 at 9:55
